packages/Datepicker/src/Datepicker.tsx

```diff
@@ -0,0 +1,158 @@
+import * as React from 'react';
+import cx from 'classnames';
+
+import Dropdown from '@igloo-ui/dropdown';
+import Input from '@igloo-ui/input';
+import Button from '@igloo-ui/button';
+import IconCalendar from '@igloo-ui/icons/dist/Calendar';
+
+import {
+  getLocalTimeZone,
+  now,
+  parseAbsoluteToLocal,
+  ZonedDateTime,
+} from '@internationalized/date';
+
+import Calendar from './components/Calendar';
+
+import './datepicker.scss';
+
+type Date = { utc: string; local: string };
+
+export interface DatepickerProps {
+  /** Selected value for the date picker. */
+  selectedDay?: string;
+  /** Specifies the value inside the input. */
+  value?: string;
+  /** Text that appears in the form control when it has no value set. */
+  placeholder?: string;
+  /** Defines a string value that labels the current element. */
+  ariaLabel?: string;
+  /** True if the date picker should be disabled. */
+  disabled?: boolean;
+  /** Form.ValidatedField state. True if it has an error. */
+  error?: boolean;
+  /** True if the Dropdown list is displayed. */
+  isOpen?: boolean;
+  /** Callback function that will be called when the user types something. */
+  onChange?: (date: Date | null) => void;
+  /** Callback when the user clicks outside the Dropdown. */
+  onClose?: () => void;
+  /** Function called when the element receives focus. */
+  onFocus?: () => void;
+  /** Add a data-test tag for automated tests. */
+  dataTest?: string;
+  /** True if the control's value can be cleared. */
+  clearable?: boolean;
+  /** Label for the clear button. Required if clearable is set to True */
+  clearLabel?: string;
```

Comment:
`clearable` and `clearLabel` are way too much look-like terms and may trigger the scramble text effect (tu puex lrie ça snas prèlbome par expmele)

So maybe rename `clearable` for `isClearable` and `clearLabel` of `clearLabelText`

Reply:
Good point I will change `clearable` for`isClearable` moreover it corresponds more to the standard used in the other components. On the other hand I will not change `clearLabel` because it is clear and is the same standard of `ariaLabel`.